packages/core/rollup.config.js

```diff
@@ -1,41 +1,102 @@
-import nodeResolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
-import replace from 'rollup-plugin-replace';
-import uglify from 'rollup-plugin-uglify';
-import { list as babelHelpersList } from '@babel/helpers';
+import * as path from 'path'
+import alias from 'rollup-plugin-alias'
+import nodeResolve from 'rollup-plugin-node-resolve'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import uglify from 'rollup-plugin-uglify'
+import { rollup as lernaAlias } from 'lerna-alias'
+import pkg from './package.json'
 
-var env = process.env.NODE_ENV
-var config = {
-  output: {
-    format: 'umd',
-    name: 'ReduxSaga',
-    exports: 'named',
-  },
+const ensureArray = maybeArr => (Array.isArray(maybeArr) ? maybeArr : [maybeArr])
+
+const makeExternalPredicate = externalArr => {
+  if (!externalArr.length) {
+    return () => false
+  }
+  const pattern = new RegExp(`^(${externalArr.join('|')})($|/)`)
+  return id => pattern.test(id)
+}
+
+const deps = Object.keys(pkg.dependencies || {})
+const peerDeps = Object.keys(pkg.peerDependencies || {})
+
+const createConfig = ({ input, output, external, env, min = false }) => ({
+  input,
+  experimentalCodeSplitting: typeof input !== 'string',
+  output: ensureArray(output).map(format =>
+    Object.assign({}, format, {
+      name: 'ReduxSaga',
+      exports: 'named',
+    })
+  ),
+  external: makeExternalPredicate(external === 'peers' ? peerDeps : deps.concat(peerDeps)),
   plugins: [
+    alias(lernaAlias()),
     nodeResolve({
       jsnext: true,
     }),
     babel({
       exclude: 'node_modules/**',
-      externalHelpersWhitelist: babelHelpersList.filter(helperName => helperName !== 'AsyncGenerator'),
+      babelrcRoots: path.resolve(__dirname, '../*'),
```

Comment:
`path.resolve(__dirname, '../*')` points to the `packages/` folder. Is that the correct folder? 😕

Reply:
yeah, that was the intended directory

babel@7 stopped using babelrc files outside of current working directory (which is packages/core here) - one have to opt in to using them with `babelrcRoots` option. 

The intention of this is to always build (& test etc) from our source directories instead of some build artifacts which might be outdated at the time of running the script